### ISSUE TITLE
Section "main" does not exist

### ISSUE THREAD

**Issue by weblution** [in2code-de/powermail#974]
With following TYPO3 installation:
- TYPO3 12.4.11
- PHP 8.2
- only EXT:powermail installed
- only ts setup:
	page = PAGE
	page.10 < styles.content.get
	(and fluid_styled_content and powermail included)

I sometimes get following exception when submitting a form and then no mails are sent:

Section "main" does not exist.

Deleting the line introduced with following commit solves the problem:
https://github.com/in2code-de/powermail/commit/d7fbf5db9750eb48dc82b3b74173b9e42a2daf36

**Comment by ddoronenko** [comment]
I have the same issue.

Section "main" does not exist.

Deleting the line introduced with following commit solves the problem:
https://github.com/in2code-de/powermail/commit/d7fbf5db9750eb48dc82b3b74173b9e42a2daf36

TYPO3 12.4.11
Powermail 12.1.1
PHP 8.2

**Comment by weblution** [comment]
As a temporary workaround for TYPO3 12.4.11 I now use
```
"in2code/powermail": "12.0.*",
"symfony/expression-language" : "^6.4",
```

**Comment by mschwemer** [comment]
Thanks for your report and tests. 

It's fixed in version 12.2.0
